### ISSUE TITLE
How does this work?

### ISSUE THREAD

**Issue by ghost** [Kisspeace/NsfwBox#16]
idk if i am stupid but how does this actually work i want to download an users pics from nsfw.xxx but i cant.

**Comment by Kisspeace** [comment]
Hi, on Android you need to give file system permission

**Comment by ghost** [comment]
No im on pc

**Comment by Kisspeace** [comment]
Try double click this button, the download folder will open in explorer
![screenshot](https://github.com/Kisspeace/NsfwBox/assets/101427274/91e8db33-345d-4397-b99c-dc5ca3a22629)

**Comment by ghost** [comment]
Yeah it opens up but i think i cant do the tab thing i dont understand it

**Comment by ghost** [comment]
where do i type the nsfw.xxx users name

**Comment by Kisspeace** [comment]
To search content by username:
1) You need to open search menu (click the search button)
![1](https://github.com/Kisspeace/NsfwBox/assets/101427274/66ecfc90-5b76-4939-9c83-d4e1562fdc58)

2) Change search type
![2](https://github.com/Kisspeace/NsfwBox/assets/101427274/7af7f280-9196-47d3-b3bf-8335ee6c4cce)
![3](https://github.com/Kisspeace/NsfwBox/assets/101427274/112af6a7-f08e-43cc-90a9-ce5d19f06c3a)

3) Enter username 
![4](https://github.com/Kisspeace/NsfwBox/assets/101427274/05ac4620-ff86-4b71-9efe-21a2506b015d)

4) And click search button
![5](https://github.com/Kisspeace/NsfwBox/assets/101427274/66ecfc90-5b76-4939-9c83-d4e1562fdc58)

**Comment by ghost** [comment]
![g](https://github.com/Kisspeace/NsfwBox/assets/155374624/94f318ed-5bc5-4d41-8ad3-8fc7c7395aaa)
it says this

**Comment by Kisspeace** [comment]
Enable this option in settings and try again
![option](https://github.com/Kisspeace/NsfwBox/assets/101427274/b16f912d-9831-4c2b-9c55-c0f51c0c40c3)

**Comment by ghost** [comment]
Still error

**Comment by Kisspeace** [comment]
What version of Windows are you using?

**Comment by ghost** [comment]
win 10

**Comment by Kisspeace** [comment]
but from the browser nsfw.xxx works fine?

**Comment by thone71** [comment]
The android ver sucks, and the only premisstion i can could n gave is to see pictures.
